core/neat/mixins/_grid-container.scss

```diff
@@ -19,7 +19,7 @@
 
 @mixin grid-container($grid: $neat-grid) {
   @if $grid != $neat-grid {
-    @warn "`grid-container` does not use grid properties.
+    @warn "`grid-container` does not use grid properties. \
```

Comment:
For the sake of consistency, can we mimic [how Bourbon does it](https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/library/_contrast-switch.scss#L66-L67) and use two strings concatenated together with a `+`? Also, lining up Line 23 to 22 based on the first quotation marks?